<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessForm</class>
 <widget class="QWidget" name="ChessForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout1"/>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>334</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_pvp">
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>人人对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_pve">
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>人机对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nvn">
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>网络对战</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
